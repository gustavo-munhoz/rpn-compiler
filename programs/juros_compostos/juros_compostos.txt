(1000 MEM)
( 10 ( ( (MEM) 1.05 * ) MEM ) FOR )
(MEM)