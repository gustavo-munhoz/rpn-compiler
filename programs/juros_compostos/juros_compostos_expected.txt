0x63D0
0x665D
0x665D